<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fennetre1</class>
 <widget class="QMainWindow" name="fennetre1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fennetre1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>951</width>
      <height>611</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="formulere">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>125</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Date </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>185</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Role</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>91</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Prenom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prenom">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>91</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_role">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>185</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_AJemp">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>245</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 2px solid #808080;   /* Grey border */
    border-radius: 10px;         /* Round the corners */
    padding: 0 8px;              /* Padding inside the button */
    background: #808080;         /* Grey background */
    color: white;                /* White text color */
}

QPushButton:hover {
    background-color: #505050;   /* Darker grey on hover */
}
</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Employer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>155</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_email">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>155</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_motpasse">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>215</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>215</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Mot passe</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>99</x>
       <y>119</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_matricule">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Matricule</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Mailing">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>181</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 2px solid #808080;   /* Grey border */
    border-radius: 10px;         /* Round the corners */
    padding: 0 8px;              /* Padding inside the button */
    background: #808080;         /* Grey background */
    color: white;                /* White text color */
}

QPushButton:hover {
    background-color: #505050;   /* Darker grey on hover */
}
</string>
     </property>
     <property name="text">
      <string>Envoyer</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Mailing</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="Mailing_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>181</width>
       <height>251</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border:2px solid gray;
border-radius: 10px;
padding:0 8px;
background:gray;
}</string>
      </property>
      <property name="text">
       <string>Envoyer</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>141</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Mailing</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>760</width>
      <height>121</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>763</width>
      <height>16777215</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>Matricule</string>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>nom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prenom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>date naissance</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Adresse mail</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Role</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>mot de passe</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_suprimer">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>80</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #808080;   /* Grey border */
    border-radius: 10px;         /* Round the corners */
    padding: 0 8px;              /* Padding inside the button */
    background: #808080;         /* Grey background */
    color: white;                /* White text color */
}

QPushButton:hover {
    background-color: #505050;   /* Darker grey on hover */
}

</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modifier">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>80</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #808080;   /* Grey border */
    border-radius: 10px;         /* Round the corners */
    padding: 0 8px;              /* Padding inside the button */
    background: #808080;         /* Grey background */
    color: white;                /* White text color */
}

QPushButton:hover {
    background-color: #505050;   /* Darker grey on hover */
}
</string>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #808080;   /* Grey border */
    border-radius: 10px;         /* Round the corners */
    padding: 0 8px;              /* Padding inside the button */
    background: #808080;         /* Grey background */
    color: white;                /* White text color */
}

QPushButton:hover {
    background-color: #505050;   /* Darker grey on hover */
}
</string>
    </property>
    <property name="text">
     <string>Recherche</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_matricule_id">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>418</x>
      <y>80</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #808080;   /* Grey border */
    border-radius: 10px;         /* Round the corners */
    padding: 0 8px;              /* Padding inside the button */
    background: #808080;         /* Grey background */
    color: white;                /* White text color */
}

QPushButton:hover {
    background-color: #505050;   /* Darker grey on hover */
}
</string>
    </property>
    <property name="text">
     <string>Tri par ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #808080;   /* Grey border */
    border-radius: 10px;         /* Round the corners */
    padding: 0 8px;              /* Padding inside the button */
    background: #808080;         /* Grey background */
    color: white;                /* White text color */
}

QPushButton:hover {
    background-color: #505050;   /* Darker grey on hover */
}
</string>
    </property>
    <property name="text">
     <string>PDF</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>761</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 2px solid #003153;   /* Prussian Blue border */
    border-radius: 10px;         /* Round the corners */
    padding: 0 8px;              /* Padding inside the button */
    background: #003153;         /* Prussian Blue background */
    color: white;                /* White text color */
}

QPushButton:hover {
    background-color: #001f33; /* Darker Prussian blue on hover */
}

</string>
     </property>
     <property name="text">
      <string>Sponsor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 2px solid #003153;   /* Prussian Blue border */
    border-radius: 10px;         /* Round the corners */
    padding: 0 8px;              /* Padding inside the button */
    background: #003153;         /* Prussian Blue background */
    color: white;                /* White text color */
}

QPushButton:hover {
    background-color: #001f33; /* Darker Prussian blue on hover */
}
</string>
     </property>
     <property name="text">
      <string>Billet</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 2px solid #003153;   /* Prussian Blue border */
    border-radius: 10px;         /* Round the corners */
    padding: 0 8px;              /* Padding inside the button */
    background: #003153;         /* Prussian Blue background */
    color: white;                /* White text color */
}

QPushButton:hover {
    background-color: #001f33; /* Darker Prussian blue on hover */
}
</string>
     </property>
     <property name="text">
      <string>Employer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 2px solid #003153;   /* Prussian Blue border */
    border-radius: 10px;         /* Round the corners */
    padding: 0 8px;              /* Padding inside the button */
    background: #003153;         /* Prussian Blue background */
    color: white;                /* White text color */
}

QPushButton:hover {
    background-color: #001f33; /* Darker Prussian blue on hover */
}
</string>
     </property>
     <property name="text">
      <string>Match</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 2px solid #003153;   /* Prussian Blue border */
    border-radius: 10px;         /* Round the corners */
    padding: 0 8px;              /* Padding inside the button */
    background: #003153;         /* Prussian Blue background */
    color: white;                /* White text color */
}

QPushButton:hover {
    background-color: #001f33; /* Darker Prussian blue on hover */
}
</string>
     </property>
     <property name="text">
      <string>Equipe</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pic2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Mailing_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>250</y>
      <width>181</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Statistique</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_showEmployes">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #808080;   /* Grey border */
    border-radius: 10px;         /* Round the corners */
    padding: 0 8px;              /* Padding inside the button */
    background: #808080;         /* Grey background */
    color: white;                /* White text color */
}

QPushButton:hover {
    background-color: #505050;   /* Darker grey on hover */
}
</string>
    </property>
    <property name="text">
     <string>show</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
